### PR TITLE
Add Abine DeleteMe

### PR DESCRIPTION
**Pull request by coreystone**
Similarweb ranking: [#109,514](https://www.similarweb.com/website/joindeleteme.com/#overview)

DeleteMe is a service that acts as an authorized agent for users in filing data deletion requests with popular data brokers and people search sites.

### CODE REVIEW

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please remove omg.lol from your PR. In addition, here's an SVG you can use. 
DeleteMe

**Comment by Carlgo11** [comment]
As the requested changes haven't been resolved yet I'm going to label this PR as inactive.
> **Warning**
> Unless there's activity within the next 7 days I'll close the PR.

**Comment by coreystone** [comment]
Sorry I keep adding unrelated commits, I'm dumb and struggling to learn how to remove them from this PR. Could I close and open another?

File: entries/j/joindeleteme.com.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "DeleteMe": {
+    "domain": "joindeleteme.com",
+    "url": "https://joindeleteme.com/",
+    "img": "joindeleteme.com.png",
```
Comment (hkamran80) [inline entries/j/joindeleteme.com.json:5]:
```suggestion
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
Please remove `omg.lol` and the PNG

**Comment by hkamran80** [comment]
> Sorry I keep adding unrelated commits, I'm dumb and struggling to learn how to remove them from this PR. Could I close and open another?

Sure, go ahead

**Pull request closed without merging**